sn_networking/src/lib.rs

```diff
@@ -658,7 +658,6 @@ impl Network {
         req: &Request,
         get_all_responses: bool,
     ) -> Vec<Result<Response>> {
-        debug!("send_and_get_responses for {req:?}");
```

Comment:
I think here we need to `log` some ID stuff of the `req` , so that we can trace it down?